src/backend/app.js

```diff
@@ -21,10 +22,114 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+//app.get("/my-route", (req, res) => {
+//res.send("Hi friend");
+//});
+
+const todayDate = new Date();
+//future meals
+app.get("/future-meals", async (req, res) => {
+  try {
+    const futureMeals = await knex
+      .select("*")
+      .from("meal")
+      .where("meal_time", ">", todayDate);
+    if (futureMeals.lenght > 0) {
+      res.status(200).json(futureMeals);
+    }
+    res
+      .status(404)
+      .send("Not found");
+  } catch (err) {
+    console.error("Server Error", err);
+    res.status(500).send("Error: Unable to fetch");
+  }
+});
+
+//past meals
+app.get("/past-meals", async (req, res) => {
+  try{
+    const pastMeals = await knex
+    .select("*")
+    .from("meal")
+    .where("meal_time", "<", todayDate);
+    if (pastMeals.length > 0) {
+      res.status(200).json(pastMeals);
+    }
+    res
+    .status(404)
+    .send("Not found");
+  } catch (err) {
+    console.error("Server Error", err);
+    res.status(500).send("Error: Unable to fetch")
+  }
+})
+
+//All meals
+app.get("/all-meals", async (req, res) => {
+  try{
+  const allMeals = await knex
+  .select("*")
+  .from("meal")
+  .orderBy("id");
+  if (allMeals > 0) {
+    res.status(200).json(allMeals);
+  }
```

Comment:
it is better to handle an empty state first and then return the response with data. it is best practice. :-) 
so check if there is no data return message for instance "not found " 
and then return the response with data